api.js

```diff
@@ -1,11 +1,48 @@
+import { divCreate} from "./main.js";
+
 const shortenLinkBtn = document.querySelector('.shorten-btn');
 const linkInput = document.querySelector('.text-input');
 const shortenDisplay = document.querySelector('.link-display');
 
 
+function getShortenLink(text){
+  fetch(`https://api.shrtco.de/v2/shorten?url=${text}`)
+  .then(res => res.json())
+  .then(data => {
```

Comment:
You should get into the habit of decoupling your fetch logic from your rendering logic. Benefits of this would be a separation of concern. You can imagine that if you're working on another page or feature that requires this data from this api. Since you're fetch data is coupled with this rendering logic you're not able to do anything else with this function. Also let's say you're working with another engineer and they need the data from this api and now they have to either refactor this or create another fetch function which pollutes the code base more. 

Something like this:

async function fetchData() {
  try {
    const response = await fetch('https://api.example.com/data');
    const data = await response.json();
    return data;
  } catch (error) {
    console.log('Error:', error);
    throw error;
  }
}

fetchData()
  .then(data => {
    // Handle the returned data here
    console.log(data);
  })
  .catch(error => {
    // Handle any errors that occurred during the fetch request
    console.log(error);
  });